libs/ngrx-toolkit/package.json

```diff
@@ -9,10 +9,9 @@
   "peerDependencies": {
     "@angular/common": "^17.0.0",
     "@angular/core": "^17.0.0",
-    "@ngrx/signals": "^17.0.0"
-  },
-  "dependencies": {
-    "tslib": "^2.3.0"
+    "@ngrx/signals": "^17.0.0",
+    "@ngrx/store": "^17.0.0",
```

Comment:
Can we leave that out for now? @mikezks, I think we need to find a better solution here. It shouldn't be that ngrx/store (and rxjs) come as dependency of the toolkit.

Reply:
Fully agree. I would not put store and signals into dependencies or peerDependencies.

I may put parts of the Redux Connector into a secondary entry point later to add a peer dependency on the store, but for now I would remove them again.

Reply:
The `@nx/dependency-checks` rule which brought up those changes offers fixers for missing or unused dependencies - the result would be exactly the same as with this PR.

With Mikes potential plan ahead I'll revert this change.

Reply:
@mikezks does that actually work? 

https://github.com/testing-library/angular-testing-library/tree/main/projects/testing-library/jest-utils has the same issue with a dependency to Jest. The fixed it by not including it into the `package.json` (like we have it now).